leetcode/generate_md.py

```diff
@@ -4,16 +4,18 @@
 new = open(r"C:\Users\User\Desktop\python\linked-list.md", "a")
```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 Merge branch 'master' into bugfix 
 Mistakes: 
 Where have you found such homework(Merge)? It is not from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS'].
There is only one group(branch), how could you make mistake here? So, it is definetly not 1013.
Wait...What did you do('master')? It is not from ['Added', 'Deleted'].